### PR TITLE
Dotenv

### PR DESCRIPTION
**Pull request by loliee**
# Description

- Refactor loadConfig method
- Add dotenv support

# Depends

Break change: php-command need to call compile() on parameter bag

### CODE REVIEW

File: src/Cilex/Application.php [hunk 1]
```diff
@@ -147,4 +146,18 @@ public function command(Command $command)
...
     }
+
+    /**
+     * Load the environment parameters.
+     *
+     * Only the parameters starting with "SYMFONY__" are considered.
+     */
+    public function loadEnvParameters()
+    {
+        foreach ($_SERVER as $key => $value) {
```
Comment (CaptainJojo) [inline src/Cilex/Application.php]: c'est quoi ce $SERVER

Comment (loliee) [inline src/Cilex/Application.php]: Dans dotenv: https://github.com/vlucas/phpdotenv/blob/master/src/Loader.php#L346
From symfony: https://github.com/symfony/http-kernel/blob/199abdfab0cb904ebd612ce136ff9f47efc151f6/Kernel.php#L536

Voila ca marche comme ça les dotenv :))

ping @CaptainJojo

**Comment by JiceTe** [comment]
:ok:

**Comment by CaptainJojo** [comment]
:+1:

**Pull request merged**